lib/recipe_box_web/templates/cuisine/index.html.eex

```diff
@@ -0,0 +1,9 @@
+<h1>Cuisines</h1>
+
+<table>
+  <%= for cuisine <- @cuisines do %>
+    <tr>
+     <td><b><%= cuisine %></b></td>
```

Comment:
When I look at this template in isolation, my initial assumption is that `cuisine` is some object and consequently "expect" this line to render some weird representation of the cuisine data structure in memory (ex. something like `#<Cuisine:0x00007f7fb495e788 @name="Italian">`, if this were Ruby). It's only when I go to the controller do I understand that the variable named `cuisines` is really a list of cuisine _names_ only.

I see a few options here. The first, short-term approach would be to simply name the variable `cuisine_names`. However, it seems likely that before long you'll want/need to display more information about the cuisine on this page (ex. when these turn into links they'll presumably need the id). Therefore the second option would be to pass a list of cuisine objects to the view and call attributes on them (ex. `cuisine.name`). On the index page, you probably won't need to get much fancier than that; however, it could be a good idea to look up how to do presenters in Elixir (a little different than Ruby/Python because it's not object-oriented).